### PR TITLE
feat(docs): add locale switcher (VIV-1274)

### PR DESCRIPTION
**Pull request by RichardHelm**
When adding `locale-switcher` to preview:
```diff
```html preview locale-switcher
```

It will add a locale switcher with the example:
image

### CODE REVIEW

File: apps/docs/rollup.config.js [hunk 1]
```diff
@@ -39,10 +39,40 @@ const input = components.reduce((inputArray, componentName) => {
...
+		${locales
+			.map(
+				(localeFile) =>
+					`'${localeName(localeFile)}': ${camelCasedLocaleName(localeFile)}`
+			)
+			.join(',\n')}
+	};
+	window.setLocale = setLocale;
+`;
+
```
Comment (YonatanKra) [inline apps/docs/rollup.config.js:71]: The global scope is getting crowded :)
Please create functions by concern (inputs, locales, importsFile) for better readability.

**Review by YonatanKra — CHANGES_REQUESTED** [review]
A small change for future generations :)

File: apps/docs/rollup.config.js (hunk 1 above)
Comment (RichardHelm) [reply apps/docs/rollup.config.js]: Sure, I refactored the file

**Review by YonatanKra — APPROVED** [review]
Great stuff. Thanks!

**Pull request merged**